client/dfclient/src/text_creator.cpp

```diff
@@ -0,0 +1,12 @@
+#include "text_creator.hpp"
+
+#include "gameplay_state.hpp"
+
+TextCreator::TextCreator(Resources& resources) : _resources(resources) {
+    _textColor = ncine::Color(0, 0, 0);
+    _pos_x = 0;
+    _pos_y = 0;
+    _textScale = 1.0f;
+    _from = 255; _hold = 60; _decay = 60;
+    _layer = Layers::TEXT;
```

Comment:
You can assign such defaults in class body, right after field declaration.

E.g.:

```c++
class X {
public:
    ncine::Color color = ncine::Color(1, 2, 3);
};
```

Reply:
This is due to necessity to initialize _resources here, so I figured that all the values should be initialized in the same place. Probably not the best idea though, as it makes it less clear what the default initialized values are, when they are in another file. Will move init to .hpp.